### PR TITLE
Feat(eos_cli_config_gen): Add support for additional dot1x commands.

### LINKED ISSUE #4118: Add support additional dot1x commands
### Enhancement summary

Need to support the following CLI commands.

Switch-level:

```
dot1x
   mac-based-auth radius av-pair user-name delimiter none lowercase
```

Interface-level:

```
interface Ethernet<ID>
  dot1x aaa unresponsive phone action traffic allow
  dot1x aaa unresponsive action traffic allow vlan <VID>
  dot1x mac based access-list
```

### Which component of AVD is impacted

eos_cli_config_gen

### Use case example

802.1X configurations

### Describe the solution you would like

Extended schema to support the above CLI syntax

### Describe alternatives you have considered

eos_cli key, at global level and at interface level. Works for now, but not ideal

### Additional context

_No response_

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by laxmikantchintakindi**
## Change Summary

Add support for additional dot1x commands.

## Related Issue(s)

Fixes #4118

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes
Need to support the following CLI commands.

Switch-level:

```
dot1x
   mac-based-auth radius av-pair user-name delimiter none lowercase
```
Interface-level:

```
interface Ethernet<ID>
  dot1x aaa unresponsive phone action traffic allow
  dot1x aaa unresponsive action traffic allow vlan <VID>
  dot1x mac based access-list
```
### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [ ] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/dot1x.schema.yml [hunk 1]
```diff
@@ -33,6 +33,28 @@ keys:
             max: 300
             convert_types:
               - str
+      mac_based_auth:
+        type: dict
+        description: Dot1x mac based auth.
+        keys:
+          radius:
```
Comment (Vibhu-gslab) [inline python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/dot1x.schema.yml]: Is there any other key as well apart from radius?
If not, we might not need nested dict.. one level is enough.

Comment (laxmikantchintakindi) [reply python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/dot1x.schema.yml]: Right now only radius it available. I created inner key to accommodate  TACACS or any authentication protocol in future.

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/dot1x.md [hunk 2]
```diff
@@ -66,6 +70,18 @@
       mac_based_authentication:
         delay: <int; 0-300>
         hold_period: <int; 1-300>
+
+      # Dot1x mac based auth.
+      mac_based_auth:
+
+        # RADIUS parameters.
+        radius:
```
Comment (MaheshGSLAB) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/dot1x.md]: since key `mac_based_auth ` almost same as upper one `mac_based_authentication `  so we can have below model:
```
mac_based_auth_radius:
  delimiter: <str; "colon" | "hyphen" | "none" | "period">
  mac_string_letter_case: <str; "lowercase" | "uppercase">
```

File: python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml [hunk 3]
```diff
@@ -880,6 +883,25 @@ keys:
...
+                  description: Configure AAA timeout options.
+                  keys:
+                    eap_response:
+                      type: str
+                      description: EAP response to send.
+                      valid_values:
+                        - "success"
+                        - "disabled"
+                    action:
+                      type: dict
```
Comment (Shivani-gslab) [inline python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml:1089]: did you re-verify the timeout parameter in other EOS version?

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/ethernet-interfaces.md [hunk 4]
```diff
@@ -901,6 +923,58 @@
             authentication_failure_fallback_mba:
               enabled: <bool>
               timeout: <int; 0-65535>
+          aaa:
+
+            # Configure AAA timeout options.
+            unresponsive:
+
+              # EAP response to send.
+              eap_response: <str; "success" | "disabled">
```
Comment (MaheshGSLAB) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/ethernet-interfaces.md:1089]: make it success as default 
```
wan1-dc1(config-if-Et1)#dot1x aaa unresponsive eap response ?
  disabled  Send no response
  success   Send EAP success (default)
  <cr>  
  ```

File: python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml [hunk 5]
```diff
@@ -880,6 +883,25 @@ keys:
...
+                      $ref: "eos_cli_config_gen#/keys/dot1x/keys/aaa/keys/unresponsive/keys/action"
```
Comment (Shivani-gslab) [inline python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml:1090]: the configs in interface and global dot1x are not same (having access_list in interface),
not sure if its for specific version, please recheck.

```
s1-leaf1(config-dot1x)#aaa unresponsive action traffic allow ?
  vlan  Allow traffic in VLAN when AAA times out
  <cr>  
```


```
s1-leaf2(config-if-Et2)#dot1x aaa unresponsive action traffic allow ?
  access-list  Apply access-list when AAA times out
  vlan         Allow traffic in VLAN when AAA times out
  <cr>  
```

File: python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/dot1x.j2 [hunk 6]
```diff
@@ -99,6 +99,9 @@ dot1x
    radius av-pair framed-mtu {{ dot1x.radius_av_pair.framed_mtu }}
 {%             endif %}
 {%         endif %}
+{%         if dot1x.mac_based_auth.radius.delimiter is arista.avd.defined and dot1x.mac_based_auth.radius.mac_string_letter_case is arista.avd.defined %}
```
Comment (MaheshGSLAB) [inline python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/dot1x.j2]: you need add your condition in line number 21 as if you have to only defined mac_based_auth parameter then it will not reflect in your config. Right now you are using it in molecule and it has other parameters so its reflecting in config.

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml [hunk 7]
```diff
@@ -1023,3 +1023,20 @@ ethernet_interfaces:
       frequency_unit: ghz
       media:
         override: 100gbase-ar4
+  - name: Ethernet70
+    description: dot1x_aaa_unresponsive
+    shutdown: false
+    dot1x:
+      aaa:
+        unresponsive:
+          eap_response: success
```
Comment (MaheshGSLAB) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml:1363]: eap response config is not generated in cfg file, please take look

File: python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml (hunk 3 above)
Comment (laxmikantchintakindi) [reply python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml:1089]: @MaheshGSLAB tested this PR on lab.

File: python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml (hunk 5 above)
Comment (laxmikantchintakindi) [reply python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml:1090]: Added access-list support.

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/dot1x.md (hunk 2 above)
Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/dot1x.md]: Fixed.

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/ethernet-interfaces.md (hunk 4 above)
Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/ethernet-interfaces.md:1089]: Only mentioned in description that EOS default is success.

File: python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/dot1x.j2 (hunk 6 above)
Comment (laxmikantchintakindi) [reply python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/dot1x.j2]: Added.

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml (hunk 7 above)
Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml:1363]: Fixed.

File: python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/dot1x.schema.yml (hunk 1 above)
Comment (laxmikantchintakindi) [reply python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/dot1x.schema.yml]: Fixed. Only one level dict created for mac_based_auth_radius.

File: python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml [hunk 8]
```diff
@@ -880,6 +883,43 @@ keys:
...
+            aaa:
+              type: dict
+              keys:
+                unresponsive:
+                  type: dict
+                  description: Configure AAA timeout options.
+                  keys:
+                    eap_response:
+                      type: str
+                      description: EAP response to send. EOS default is `success`.
```
Comment (Shivani-gslab) [inline python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml:1084]:
```suggestion
                      description: EAP response to send.
                      default: "success"
```

File: python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml [hunk 9]
```diff
@@ -880,6 +883,43 @@ keys:
...
+                      valid_values:
+                        - "success"
+                        - "disabled"
+                    action:
+                      type: dict
+                      keys:
+                        apply_cached_results:
+                          type: bool
+                          $ref: "eos_cli_config_gen#/keys/dot1x/keys/aaa/keys/unresponsive/keys/action/keys/apply_cached_results"
```
Comment (Shivani-gslab) [inline python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml]: instead of adding reference for each key, you can add a ref in `action` and define only `traffic_allow_access_list ` key here

File: python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/dot1x.schema.yml [hunk 10]
```diff
@@ -33,6 +33,24 @@ keys:
             max: 300
             convert_types:
               - str
+      mac_based_auth_radius:
+        type: dict
+        description: Dot1x mac based RADIUS auth.
+        keys:
+          delimiter:
+            type: str
+            description: Delimiter to use in MAC address string.
```
Comment (Shivani-gslab) [inline python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/dot1x.schema.yml:42]:
```suggestion
            description: Delimiter to use in MAC address string.
            required: true
```

File: python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/dot1x.schema.yml [hunk 11]
```diff
@@ -33,6 +33,24 @@ keys:
...
+            valid_values:
+              - colon
+              - hyphen
+              - none
+              - period
+          mac_string_letter_case:
+            type: str
+            description: MAC address string in lowercase/uppercase.
```
Comment (Shivani-gslab) [inline python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/dot1x.schema.yml:51]:
```suggestion
            description: MAC address string in lowercase/uppercase.
            required: true
```

File: python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2 [hunk 12]
```diff
@@ -225,6 +225,46 @@ interface {{ ethernet_interface.name }}
...
+{%                                     set action_apply_config = action_apply_config ~ " timeout " ~ ethernet_interface.dot1x.aaa.unresponsive[action.name].cached_results_timeout.time_duration ~ " " ~ ethernet_interface.dot1x.aaa.unresponsive[action.name].cached_results_timeout.time_duration_unit %}
+{%                                 endif %}
+{%                             endif %}
+{%                             if ethernet_interface.dot1x.aaa.unresponsive[action.name].traffic_allow is arista.avd.defined(true) %}
+{%                                 set traffic = "traffic allow" %}
+{%                             elif ethernet_interface.dot1x.aaa.unresponsive[action.name].traffic_allow_vlan is arista.avd.defined %}
+{%                                 set traffic = "traffic allow vlan " ~ ethernet_interface.dot1x.aaa.unresponsive[action.name].traffic_allow_vlan %}
+{%                             elif ethernet_interface.dot1x.aaa.unresponsive[action.name].traffic_allow_access_list is arista.avd.defined %}
```
Comment (Shivani-gslab) [inline python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2]: this should not be in elif - we can have both access_list and vlan together 

if we give `traffic_allow_vlan` and `traffic_allow_access_list` both it should concatenate and return a config like -
`dot1x aaa unresponsive action traffic allow vlan 13 access-list ac1`

**Review by Shivani-gslab — CHANGES_REQUESTED** [review]

File: python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml (hunk 5 above)
Comment (Shivani-gslab) [reply python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml:1090]: https://github.com/aristanetworks/avd/pull/4191#discussion_r1682265067

File: python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml (hunk 8 above)
Comment (laxmikantchintakindi) [reply python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml:1084]: Generally we dont add default in schema for eos defaults.

Comment (Shivani-gslab) [reply python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml:1084]: yeah, but here I think its better to add in default than description and the extension also takes default from schema.
Let's confirm with @ClausHolbechArista and @gmuloc  once.

File: python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml (hunk 9 above)
Comment (laxmikantchintakindi) [reply python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml]: Fixed.

File: python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/dot1x.schema.yml (hunk 10 above)
Comment (laxmikantchintakindi) [reply python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/dot1x.schema.yml:42]: Fixed.

File: python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/dot1x.schema.yml (hunk 11 above)
Comment (laxmikantchintakindi) [reply python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/dot1x.schema.yml:51]: Fixed.

File: python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2 (hunk 12 above)
Comment (laxmikantchintakindi) [reply python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2]: Fixed.

**Review by MaheshGSLAB — APPROVED** [review]

File: python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml (hunk 8 above)
Comment (ClausHolbechArista) [reply python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml:1084]: Laxmikant is correct here, we should not implement default values in the cli_gen schema.

**Review by Shivani-gslab — COMMENTED** [review]
Please check for coverage improvement for `ethernet_interfaces.dot1x` in this PR

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml [hunk 13]
```diff
@@ -1023,3 +1023,21 @@ ethernet_interfaces:
       frequency_unit: ghz
       media:
         override: 100gbase-ar4
+  - name: Ethernet70
```
Comment (Shivani-gslab) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml:1357]: please add other test-cases also to resolve the missing coverage

**Review by Shivani-gslab — CHANGES_REQUESTED** [review]

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml (hunk 13 above)
Comment (Shivani-gslab) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml:1357]: fixed some warnings

**Review by Shivani-gslab — APPROVED** [review]
LGTM!

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml [hunk 14]
```diff
@@ -620,7 +633,6 @@ ethernet_interfaces:
       pae:
         mode: authenticator
       authentication_failure:
-        action: allow
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml]: why is this removed?

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml [hunk 15]
```diff
@@ -630,9 +642,7 @@ ethernet_interfaces:
       timeout:
         idle_host: 10
         quiet_period: 10
-        reauth_period: server
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml]: why is this removed?

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml [hunk 16]
```diff
@@ -630,9 +642,7 @@ ethernet_interfaces:
...
         reauth_timeout_ignore: true
-        tx_period: 10
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml]: why is this removed?

File: python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2 [hunk 17]
```diff
@@ -225,6 +225,50 @@ interface {{ ethernet_interface.name }}
    dot1x authentication failure action traffic drop
 {%             endif %}
 {%         endif %}
+{%         if ethernet_interface.dot1x.aaa.unresponsive is arista.avd.defined %}
+{%             set aaa_config = "dot1x aaa unresponsive" %}
+{%             if ethernet_interface.dot1x.aaa.unresponsive.phone_action is arista.avd.defined or ethernet_interface.dot1x.aaa.unresponsive.action is arista.avd.defined %}
+{%                 set actions = [{'name': 'phone_action', 'config': aaa_config ~ ' phone action'}, {'name': 'action', 'config': aaa_config ~ ' action'}] %}
```
Comment (ClausHolbechArista) [inline python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2]: This logic make the rest pretty hard to follow. Can we simplify this somehow?
Maybe you could set a tmp variable with action_settings and parse those below - making everything shorter and more readable below.

File: python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2 [hunk 18]
```diff
@@ -225,6 +225,50 @@ interface {{ ethernet_interface.name }}
...
+                              ethernet_interface.dot1x.aaa.unresponsive[action.name].traffic_allow_access_list is arista.avd.defined %}
+{%                             if ethernet_interface.dot1x.aaa.unresponsive[action.name].apply_cached_results is arista.avd.defined(true) %}
+{%                                 set action_apply_config = "apply cached-results" %}
+{%                                 if ethernet_interface.dot1x.aaa.unresponsive[action.name].cached_results_timeout.time_duration is arista.avd.defined and ethernet_interface.dot1x.aaa.unresponsive[action.name].cached_results_timeout.time_duration_unit is arista.avd.defined %}
+{%                                     set action_apply_config = action_apply_config ~ " timeout " ~ ethernet_interface.dot1x.aaa.unresponsive[action.name].cached_results_timeout.time_duration ~ " " ~ ethernet_interface.dot1x.aaa.unresponsive[action.name].cached_results_timeout.time_duration_unit %}
+{%                                 endif %}
+{%                             endif %}
+{%                             if ethernet_interface.dot1x.aaa.unresponsive[action.name].traffic_allow is arista.avd.defined(true) or
```
Comment (ClausHolbechArista) [inline python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2]: I think this `if` just contributes to cluttering the template. Please look into simplifying the nesting. It is ok if we have to give the `traffic allow` part of the string in multiple places.

File: python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/dot1x.j2 [hunk 19]
```diff
@@ -99,6 +99,9 @@ dot1x
    radius av-pair framed-mtu {{ dot1x.radius_av_pair.framed_mtu }}
 {%             endif %}
 {%         endif %}
+{%         if dot1x.mac_based_auth_radius.delimiter is arista.avd.defined and dot1x.mac_based_auth_radius.mac_string_letter_case is arista.avd.defined %}
```
Comment (ClausHolbechArista) [inline python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/dot1x.j2]: The CLI is very different than the variable. I think you should change the model to be `av_pair_user_name_delimiter` since there could be other options coming later.

Comment (ClausHolbechArista) [reply python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/dot1x.j2]: same for case. Maybe find a better variable name there too. `av_pair_user_name_case` maybe? (please check wording on EOS)

**Review by ClausHolbechArista — CHANGES_REQUESTED** [review]

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml (hunk 14 above)
Comment (Shivani-gslab) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml]: The test for this already exists at other place in this file and there was a warning in coverage report for it being always true.
That's why changed it to reduce the warnings in report.

same for the below 2 `reauth_period` and `tx_period`

File: python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2 (hunk 17 above)
Comment (laxmikantchintakindi) [reply python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2]: Simplified the template.

File: python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/dot1x.j2 (hunk 19 above)
Comment (laxmikantchintakindi) [reply python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/dot1x.j2]: I updated the schema as below:
radius_av_pair_username_format:
  delimiter:
  mac_string_case:
  
  EOS shows below description for lowercase/uppercase setting.
  
  s1-leaf2(config-dot1x)#mac-based-auth radius av-pair user-name delimiter colon  ?
  lowercase  MAC address string in lowercase
  uppercase  MAC address string in uppercase

File: python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2 (hunk 18 above)
Comment (laxmikantchintakindi) [reply python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2]: Fixed.

File: python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2 [hunk 20]
```diff
@@ -225,6 +225,58 @@ interface {{ ethernet_interface.name }}
    dot1x authentication failure action traffic drop
 {%             endif %}
 {%         endif %}
+{%         if ethernet_interface.dot1x.aaa.unresponsive is arista.avd.defined %}
```
Comment (Vibhu-gslab) [inline python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2:372]: if we can keep `ethernet_interface.dot1x.aaa.unresponsive` in a variable then this template will become much easier to read.

Comment (laxmikantchintakindi) [reply python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2:372]: Done.

**Review by ClausHolbechArista — APPROVED** [review]
